packages/common/hooks/useDocsSearch.ts

```diff
@@ -199,24 +198,26 @@ const useDocsSearch = () => {
 
       let sourcesLoaded = 0
 
-      supabaseClient.rpc('docs_search_fts', { query: query.trim() }).then(({ data, error }) => {
-        sourcesLoaded += 1
-        if (error || !Array.isArray(data)) {
-          dispatch({
-            type: 'errored',
-            key: localKey,
-            sourcesLoaded,
-            message: error?.message ?? '',
-          })
-        } else {
-          dispatch({
-            type: 'resultsReturned',
-            key: localKey,
-            sourcesLoaded,
-            results: data,
-          })
-        }
-      })
+      supabaseClient
+        .rpc('docs_search_fts', { query: query.trim() })
+        .then(({ data, error }: { data: any; error: any }) => {
```

Comment:
Only change is the types here. These used to be implicit `any` anyway, but tests will fail with TS complaint if we don't define it explicitly now.